src/message_sender.c

```diff
@@ -361,38 +361,46 @@ static SEND_ONE_MESSAGE_RESULT send_one_message(MESSAGE_SENDER_INSTANCE* message
                 }
                 else
                 {
-                    for (i = 0; i < body_data_count; i++)
+                    if (body_data_count <= 0)
```

Comment:
>< [](start = 40, length = 1)

body_data_count is size_t. It cannot be negative per C standard.

Reply:
Updated the PR...